pkg/registry/core/service/storage/rest_test.go

```diff
@@ -2374,7 +2358,25 @@ func TestUpdateNodePorts(t *testing.T) {
 
 func TestServiceRegistryCreateWithNetworkOfExternalIPAM(t *testing.T) {
 	storage, registry, server := NewTestREST(t, nil)
-	storage.networks = &dummyNetworkGetter{ipamType: "External"}
+	storage.fnGetNetwork = func(ctx context.Context, tenant, name string) (object runtime.Object, err error) {
+		return &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": "arktos.futurewei.com/v1",
```

Comment:
SO many hard-coded values. It is better to define some vars and consts.

Reply:
good point. will use consts..